lib/index.js

```diff
@@ -44,6 +44,14 @@ module.exports = function (gulp, options) {
     mocha: {}
   });
 
+  // allows command line arguments to modify the options object
+  // ex: gulp test --mocha.grep @production-only --mocha.invert
+  // -- options.mocha == { grep: '@production-only', invert: true }
+  _.each(argv, (value, name) => {
+    if (name === '_' || name === '$0') return;
+    options[name] = _.merge(options[name] || {}, value);
```

Comment:
This is only going one level deep; it'd be a more complete feature if you could override options at any level. Also, it'd be useful to have this command line override ability documented in `README.md`.